src/Forms/__tests__/Form.test.tsx

```diff
@@ -0,0 +1,40 @@
+import { Button } from 'Buttons';
+import { Form, TextInput } from 'Forms';
+import React from 'react';
+import {
+  render,
+  BoundFunction,
+  GetByBoundAttribute,
+} from 'test-utils';
+import { VARIANT } from 'Theme';
+
+describe('Form', function () {
+  let getByTestId: BoundFunction<GetByBoundAttribute>;
+  const formTestId = 'testForm';
+  beforeEach(function () {
+    ({ getByTestId } = render(
+      <Form
+        id="testForm"
+        dataTestId={formTestId}
+      >
+        <TextInput
+          id="testSemester"
+          name="semester"
+          value="Spring"
+          errorMessage="Error: Please enter a valid ID"
+          label="semester"
+          onChange={() => {}}
+        />
+        <Button
+          onClick={() => {}}
+          variant={VARIANT.BASE}
+        >
+          Submit
+        </Button>
+      </Form>
+    ));
+  });
+  it('renders', function () {
+    getByTestId(formTestId);
```

Comment:
Ah right - I had used the `test-id` as a final resort because the [ARIA form role docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Form_Role) were saying to add the `role` of `form` when not using `<form>` (e.g. if you're using `<div>` instead of `form`) and that using `<form>` was sufficient for letting screen readers know that it is a form. Adding a role of form does work, but I noticed when looking at the docs that we should be adding an `aria-label` to forms to distinguish their purposes - I replaced the test id property with the `label` property in 93c724b. This way, I can use `getByLabelText` in the test instead of `getByTestId.`

Reply:
That link is specific to using `role="form"` on a non-`<form>` element, which we are correctly not doing. But the `ByRole` queries in `testing-library` also work with implicit accessibility roles, and because the `<form>` element does have an implicit role of `form'`, we can use `getByRole('form')` and it will correctly query for the `<form>` element and confirm that our `Form` has been rendered.

Reply:
Ok thanks! This was my original approach (to do `getByRole`), but I must have done it wrong initially, because I was getting an error saying that the role 'form' couldn't be found.. so I probably had a typo or something the first time around, which sent me down this rabbit hole! I updated the test to use `getByRole` in commit aadb274.